### PR TITLE
enh(Resource-status/Listing): Improve Listing responsiveness

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This takes advantage of the new Listing layout to make the columns more responsive, and prevent them from wrapping around:

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

**Review by kduret — COMMENTED** [review]
remove `centreon-license-manager/.DS_Store` file

File: package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.10",
     "@material-ui/styles": "^4.11.2",
-    "@visx/visx": "^1.3.0",
+    "@visx/visx": "1.4.0",
```
Comment (bdauria) [inline package.json:91]: Latest visx version looks broken, fix it for the moment

File: www/front_src/src/Resources/Listing/columns/index.tsx [hunk 2]
```diff
@@ -210,6 +215,7 @@ const ParentResourceColumn = ({
 };
 
 interface ColumnsProps {
+  isSmallScreen?: boolean;
```
Comment (kduret) [inline www/front_src/src/Resources/Listing/columns/index.tsx:218]: seems not used

**Review by Thebarda — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by Thebarda — DISMISSED** [review]
Tested and LGTM

File: www/front_src/src/Resources/Listing/index.test.tsx [hunk 3]
```diff
@@ -371,7 +373,9 @@ describe(Listing, () => {
 
     const chipLabel = `${acknowledgedEntity?.name} ${labelAcknowledged}`;
 
-    const chip = await findByLabelText(chipLabel);
+    const chip = await findByLabelText(chipLabel, undefined, {
+      timeout: 10000,
```
Comment (Thebarda) [inline www/front_src/src/Resources/Listing/index.test.tsx:377]: Jest tears down the test after 5 seconds, maybe the timeout for 10 seconds is not very relevant

Comment (bdauria) [reply www/front_src/src/Resources/Listing/index.test.tsx:377]: We changed the default timeout to 10sec to avoid ci failures

**Review by Thebarda — APPROVED** [review]

File: www/front_src/src/Resources/Listing/index.test.tsx (hunk 3 above)
Comment (jdelpierre) [reply www/front_src/src/Resources/Listing/index.test.tsx:377]: why only the CI?  and why just x2 from the default?

**Review by jdelpierre — APPROVED** [review]

File: www/front_src/src/Resources/Listing/index.test.tsx (hunk 3 above)
Comment (bdauria) [reply www/front_src/src/Resources/Listing/index.test.tsx:377]: It’s not just a change within the CI, it’s global, but the failures manifested themselves mostly on the CI

**Pull request merged**